src/main/java/kitchenpos/order/application/OrderService.java

```diff
@@ -65,7 +70,20 @@ public OrderResponse changeOrderStatus(final Long orderId, final OrderChangeRequ
 
     private List<OrderLineItem> extractOrderLineItemFrom(final List<OrderLineItemDto> orderLineItemsDto) {
         return orderLineItemsDto.stream()
-                .map(OrderLineItemDto::toOrderLineItem)
+                .map(orderLineItemDto -> {
+                    Menu menu = getMenu(orderLineItemDto);
+                    return new OrderLineItem(
+                            orderLineItemDto.getMenuId(),
+                            orderLineItemDto.getMenuId(),
+                            menu.getName(),
+                            menu.getPrice(),
+                            orderLineItemDto.getQuantity()
+                    );
+                })
                 .collect(Collectors.toList());
```

Comment:
스트림 파이프라인 내부의 로직은 private 메서드 등으로 분리하는게 더 가독성이 높을 것 같아요.

```suggestion
   private List<OrderLineItem> extractOrderLineItemFrom(final List<OrderLineItemDto> orderLineItemsDto) {
        return orderLineItemsDto.stream()
                .map(this::toOrderLineItem)
                .collect(Collectors.toList());
```